.travis.yml

```diff
@@ -30,25 +30,20 @@ jobs:
     - stage: test
       name: lint
       script: npm run lint
-    - stage: test
```

Comment:
you still need this stage

see warnings: https://app.travis-ci.com/github/symbol/symbol-explorer/builds/242968726/config

Reply:
right, I thought it can be group. let me fix.